Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/GeometricFillTool.java

```diff
@@ -168,6 +189,7 @@ public void onSaveInstanceState(Bundle bundle) {
 
 		bundle.putSerializable(BUNDLE_BASE_SHAPE, baseShape);
 		bundle.putSerializable(BUNDLE_SHAPE_DRAW_TYPE, shapeDrawType);
+		bundle.putSerializable(BUNDLE_OUTLINE_WIDTH, shapeOutlineWidth);
```

Comment:
Use [Bundle.putInt(String, int)](https://developer.android.com/reference/android/os/BaseBundle.html#putInt(java.lang.String,%20int)) instead, no need to serialize a base type.